src/components/Table/utils/TableState.ts

```diff
@@ -226,15 +226,13 @@ export class TableState {
       // On initial render the columns we start with is whatever is in local storage.
       // On any subsequent render, we start with an empty array.
       // (We'll add to this after figuring out which columns are new vs existing.)
-      const columnIdsToExpand = isInitial ? localStorageColumns : [];
-
-      // Create a list of all existing column ids. (We ignore the `initExpanded` property for existing columns)
-      const existingColumnIds = this.columns.map((c) => c.id);
```

Comment:
**question:** Is there a reason why we are no longer ignoring the `initExpanded` for existing columns? The idea behind this was that if a user collapsed a column, then the columns rerender we wouldn't want to expand it s the user explicitly collapsed it.

Reply:
No, there is no reason for that change, I'll leave it as it was